conda/recipes/ucxx/meta.yaml

```diff
@@ -59,7 +64,10 @@ outputs:
       host:
         - ucx
       run:
-        - cudatoolkit {{ cuda_spec }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% endif %}
+        - cuda-version {{ cuda_spec }}
```

Comment:
Remove this as a run dependency, per https://github.com/rapidsai/ucxx/pull/61/files#r1244290996
```suggestion
```